components/Questionnaire/index.vue

```diff
@@ -219,6 +219,33 @@ export default {
         this.showLevelTwo = true
       }
     },
+    onPreviousQuestionnaire () {
+      switch (this.params.level) {
+        case (1) : {
+          this.showModalLevelDesa = false
+          this.isLevelOne = true
+          this.showLevelOne = true
+          break
+        }
+        case (2) : {
+          this.showModalLevelDesa = false
+          this.isLevelTwo = true
+          this.showLevelTwo = true
+          break
+        }
+        case (3) : {
+          this.showModalLevelDesa = false
+          this.isLevelThree = true
+          this.showLevelThree = true
+          break
+        }
+        case (4) : {
+          this.showModalLevelDesa = false
+          this.isLevelFour = true
+          this.showLevelThree = true
+        }
+      }
+    },
```

Comment:
hello, @doohanas I just wondering if `this.params.levels` is set to 1, then the `this.isLevelOne` and `this.showLevelOne` is set to `true` 

then if we change `this.params.levels` to 2, should the previous value be set to `false`, or does it not matter?